Editor/AGS.Editor/TextProcessing/SpeechLinesNumbering.cs

```diff
@@ -72,6 +72,18 @@ private void PerformNumbering(Game game)
 				game.GlobalMessages[i] = processor.ProcessText(game.GlobalMessages[i], GameTextType.Message, Character.NARRATOR_CHARACTER_ID);
 			}
 
+            if (_errors.Count > 0)

```

Comment:
Similarily as above, it's better to use `_errors.HasErrors` here, as the list may contain other messages.